component/admin/sql/install/mysql/install.sql

```diff
@@ -9,19 +49,25 @@ CREATE TABLE IF NOT EXISTS `#__redcore_oauth_clients` (
   `scope` TEXT,
   `user_id` varchar(80),
   PRIMARY KEY (`id`),
-  KEY `idx_client_id` (`client_id`)
-) DEFAULT CHARSET = utf8;
+  UNIQUE KEY `idx_client_id` (`client_id`)
```

Comment:
I noticed that all these unique/index/primary key changes are not in the upgrade and they should be to avoid inconsistent databases
When dropping the old ones however, you might want to use the same method I did - using a stored procedure to check if they're there before dropping them - since databases are always altered, and it gets worse with a product with a certain number of installs like redCORE